tests/rptest/utils/audit_schemas.py

```diff
@@ -1,463 +0,0 @@
-# Copyright 2023 Redpanda Data, Inc.
```

Comment:
Do we want to keep this? It validates what we expect to receive

Reply:
I'm removing this due to the introduction of the OCSF server that will validate the generate messages against the schema.